ResourceManager/CommitDataRenders/CommitDataBodyRenderer.cs

```diff
@@ -40,7 +40,7 @@ public string Render(CommitData commitData, bool showRevisionsAsLinks)
                 throw new ArgumentNullException(nameof(commitData));
             }
 
-            var body = "\n" + WebUtility.HtmlEncode((commitData.Body ?? "").Trim());
+            var body = WebUtility.HtmlEncode((commitData.Body ?? "").Trim());
```

Comment:
Nice! When copying text out of this box, or giving it input focus, it always felt weird having a blank line at the top here 😄

Reply:
Well, it gets better. Now it is possible to click, <kbd>CTRL + A</kbd>, <kbd>CTRL + C</kbd> the message!